eng/InstallRuntimes.proj

```diff
@@ -155,7 +160,7 @@
   <AspNetCoreVersion31>$(MicrosoftAspNetCoreApp31Version)</AspNetCoreVersion31>
 
   <RuntimeVersionLatest>$(RuntimeVersionLatest)</RuntimeVersionLatest>
-  <AspNetCoreVersionLatest>$(MicrosoftAspNetCoreAppRefVersion)</AspNetCoreVersionLatest>
+  <AspNetCoreVersionLatest>$(MicrosoftAspNetCoreAppRefInternalVersion)</AspNetCoreVersionLatest>
```

Comment:
I think this should remain `MicrosoftAspNetCoreAppRefVersion`, since this is the framework version and not the "unstabilized build" version.

Reply:
It will be the stable version i.e. 5.0.0 when everything is working. I set it to the zip file version in versions.props

    <MicrosoftAspNetCoreAppRefInternalVersion>5.0.0-rc.2.20475.17</MicrosoftAspNetCoreAppRefInternalVersion>
    <MicrosoftAspNetCoreAppRefVersion>5.0.0-rc.2.20475.17</MicrosoftAspNetCoreAppRefVersion>

Reply:
In the manifest you need the stable version (`MicrosoftAspNetCoreAppRefVersion`), right? Or is this not what you use for the exec flag?

Reply:
You are using `MicrosoftAspNetCoreAppRefInternalVersion` which will never be stable. This should be changed to `MicrosoftAspNetCoreAppRefVersion`

Reply:
Yes, you are right. Thanks.